Dockerfile-runtime

```diff
@@ -0,0 +1,34 @@
+FROM ubuntu:20.04 as build
+
+RUN apt-get update && apt-get -y install curl git dirmngr apt-transport-https lsb-release ca-certificates bash unzip
+RUN curl -sL https://deb.nodesource.com/setup_12.x | bash -
+RUN apt-get -y install nodejs gcc g++ make 
```

Comment:
Why not base yourself on the image node:12 (it's basically based on [debian:buster](https://hub.docker.com/_/debian)) or something similar for the build step? It would avoid you to install node, yarn, and all their dependencies

Reply:
This PR is based on the [Dockerfile](https://github.com/botpress/runtime/blob/master/Dockerfile) in the runtime repository. This was used for all the tests so switching from `ubuntu` to `debian` could result in weird edge case I don't think it is worth the risk.